instrumentation/spring/spring-cloud-aws-3.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("io.awspring.cloud")
+    module.set("spring-cloud-aws-sqs")
+    versions.set("(,)")
```

Comment:
Is better to use `versions.set("[3.0.0,)")` or add necessary comment to explain the plugin starting from 3.0.0 here?